R/accounts.R

```diff
@@ -98,7 +83,7 @@ connectApiUser <- function(account = NULL, server = NULL, apiKey = NULL, quiet =
   # write the user info
   registerUserApiKey(serverName = target$name,
                     accountName = account,
-                    userId = userId,
```

Comment:
I think the intent here was to pick up the userId from disk if it was present there, and to ask the server otherwise. It is probably okay to just use the value from the server all the time though!

Reply:
👍